ci/docker/runtime_functions.sh

```diff
@@ -31,14 +31,49 @@ clean_repo() {
     git submodule update --init --recursive
 }
 
+# wrap compiler calls with ccache
```

Comment:
Please add more documentation. To me, it's not clear what exactly is happening here and why we have to create these files in our source directory

Reply:
Also, is this only required before making make-calls? It seems like you're not using it for cmake. So maybe rename the function into something that makes it clear that this is a make-specific thing

Reply:
There is a comment about this 2 lines below. For cmake it is not nessesary